compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/model/BIRNode.java

```diff
@@ -483,7 +490,7 @@ public String toString() {
 
         @Override
         public Name getName() {
```

Comment:
Don't we need to have two functions to get name and displayName?

Reply:
Or this should be getDisplayName

Reply:
`getName` is inherited though `NamedNode` so we can't change that name.

but we can rename `name` to `internalName` and `displayName` to `name`.

Reply:
fixed in b80de14